model/sanbox.rb

```diff
@@ -1,13 +1,13 @@
-class X
```

Comment:
It would be better if we add this file to `.gitignore` too. We don't need it. It makes our directory cleaner